src/Accelerators/NNPA/Conversion/ZHighToZLow/ZHighToZLow.cpp

```diff
@@ -410,6 +415,9 @@ ZMemRefType convertZTensorToMemRefType(OpBuilder b, Type type) {
         h = (b.getAffineDimExpr(e1) +
              pad_size * (b.getAffineDimExpr(e1).floorDiv(constExprS)))
                 .floorDiv(constExpr64);
+#if 1
```

Comment:
Remove

Reply:
It was temporal fix for WIP draft. This part has been fixed by more logical way. Thank you.